data/tr/examples/1.txt

```diff
@@ -0,0 +1,5 @@
+!morphology
+adam yedi
+adamlar yedi
+adamlar elma yedi
+adamlar elma yediler
```

Comment:
examples should probably be moved to `corpus-basic.batch` which is where the other languages have demo text.